### PR TITLE
Add python script to configure local satellite installation

### PR DESCRIPTION
**Pull request by dav1x**
#### What does this PR do?
-Add python script to configure sat6 for disconnected installation

#### How should this be manually tested?
./ocp36-sat6.py --password admin_password

#### Is there a relevant Issue open for this?
NA
#### Who would you like to review this?
cc: @tomassedovic @cwilkers @rlopez133 @e-minguez  PTAL

### CODE REVIEW

File: reference-architecture/day2ops/scripts/ocp36-sat6.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+            self.openshift3Images.append(nl)
+
+    def _parseCli(self):
+
+        parser = argparse.ArgumentParser(description='Add all OCP images for disconnected installation to satellite 6', add_help=True)
+        parser.add_argument('--orgid', action='store', default='1',help='Satellite organization ID to create new product for OCP images in')
+        parser.add_argument('--productname', action='store', default='ocp36',help='Satellite product name to use to create OCP images')
+        parser.add_argument('--password', action='store', help='Password for hammer CLI')
+        self.args = parser.parse_args()
+
```
Comment (e-minguez) [inline reference-architecture/day2ops/scripts/ocp36-sat6.py]: I'd say to add --satellite to specify the satellite URL and --username for the user name, and if they are not specified, use the hammer defaults.
I'd also say it requires hammer to be installed in the host running the script somehow in the help

File: reference-architecture/day2ops/scripts/ocp36-sat6.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+        if not self.args.password:
+            self.args.password = getpass.getpass(prompt='Please enter the password to use for the admin account in hammer CLI: ')
+
+    def _syncData(self):
+        print "Sync repo data? (This may take a while)"
+        go = raw_input("Continue? y/n:\n")
```
Comment (e-minguez) [inline reference-architecture/day2ops/scripts/ocp36-sat6.py]: A "-y" flag can be also helpful to run it without asking anything to the user.

File: reference-architecture/day2ops/scripts/ocp36-sat6.py (hunk 1 above)
Comment (dav1x) [reply reference-architecture/day2ops/scripts/ocp36-sat6.py]: The intent was to run it locally on a satellite server. But, I can add it as an argument and default it to localhost. The username if not specified defaults to admin. But, I can change that to the same as the server.

File: reference-architecture/day2ops/scripts/ocp36-sat6.py (hunk 2 above)
Comment (dav1x) [reply reference-architecture/day2ops/scripts/ocp36-sat6.py]: Sure. That works

File: reference-architecture/day2ops/scripts/ocp36-sat6.py (hunk 1 above)
Comment (dav1x) [reply reference-architecture/day2ops/scripts/ocp36-sat6.py]:
```
[root@sat6 ~]# hammer --server localhost --password 100Root- organization list
Error: undefined method `request' for #<String:0x0000000231f1f8>
```
I'm going to add username and wait on --server

File: reference-architecture/day2ops/scripts/ocp36-sat6.py [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+            self.openshift3Images.append(nl)
+
+    def _parseCli(self):
+
+        parser = argparse.ArgumentParser(description='Add all OCP images for disconnected installation to satellite 6', add_help=True)
+        parser.add_argument('--orgid', action='store', default='1',help='Satellite organization ID to create new product for OCP images in')
+        parser.add_argument('--productname', action='store', default='ocp36',help='Satellite product name to use to create OCP images')
+        parser.add_argument('--username', action='store', default='admin', help='Satellite 6 username for hammer CLI')
+        parser.add_argument('--password', action='store', help='Satellite 6 Password for hammer CLI')
+        parser.add_argument('--no_confirm', action='store_true', help='Satellite 6 Password for hammer CLI')
```
Comment (e-minguez) [inline reference-architecture/day2ops/scripts/ocp36-sat6.py]: This help message should say "Do not ask for confirmation"

File: reference-architecture/day2ops/scripts/ocp36-sat6.py [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+                exit(0)
+
+        print "Adding product with name: %s" % self.args.productname
+
+        cmd="hammer --username %s --password %s product create --name %s --organization-id %s" % (self.args.username, self.args.password, self.args.productname, self.args.orgid)
+        os.system(cmd)
+
+        print "Adding openshift3 images"
+        for image in self.openshift3Images:
+            cmd='hammer --username %s --password %s repository create --name %s --organization-id %s --content-type docker --url "https://registry.access.redhat.com" --docker-upstream-name %s --product %s' % (self.args.username,  self.args.password, image, self.args.orgid, image, self.args.productname )
```
Comment (e-minguez) [inline reference-architecture/day2ops/scripts/ocp36-sat6.py]: This create a repo for every image. I have 0 idea about the docker stuff in satellite, but it won't be better to create a single repo with all the images? (As I said, I have 0 idea, so maybe I'm talking nonsense)

Comment (dav1x) [reply reference-architecture/day2ops/scripts/ocp36-sat6.py]: You cannot do that. It's one repo per container image.

**Pull request merged**